packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -254,11 +255,84 @@ The requested scope is invalid:
 ```
 
+### /device/authorize
+
+<ApiOperation method="post" url="${baseUrl}/v1/device/authorize" /><ApiLifecycle access="ea" />
+
+This endpoint returns user code, device code, activation link, and a QR code activation link.
+
+>  **Note:** This endpoint's base URL varies depending on whether you are using a Custom Authorization Server. For more information, see [Composing your base URL](#composing-your-base-url).
+
+#### Request parameters
+
+The following parameters can be posted as a part of the URL-encoded form values to the API.
+
+> **Note:** The `/device/authorize` endpoint requires client authentication. See the [Client authentication methods](#client-authentication-methods) section for more information on which method to choose and how to use the parameters in your request.
+
+| Parameter               | Description                                                                                                                                                                                                                                                                                                                        | Type   |
+| :---------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :----- |
+| client_id               | Obtained during either manual client registration or through the [Dynamic Client Registration API](/docs/reference/api/oauth-clients/). It identifies the client and must match the value preregistered in Okta.                                                                                                                       | String |
+| scope                   | A list of scopes that the client wants included in the access token. See [scopes](#access-token-scopes-and-claims).                                                                                                                                                                                                 | String |
+
+#### Response properties
+
+Based on the scopes requested. Generally speaking, the scopes specified in a request are included in the access token in the response.
+
+| Property                  | Description                                                                                    | Type    |
+| :------------------------ | :--------------------------------------------------------------------------------------------- | :------ |
+| verification_uri          | The verification URI that the resource owner must visit to authenticate.                            | String  |
+| verification_uri_complete | The complete verification URI that the resource owner can alternatively visit to authenticate.      | String  |
```

Comment:
A URI that includes the "user_code" that the end-user alternatively visits to verify.